File: examples/petstore/oseg_generated/csharp/FindPetsByTags_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPITools.Api;
using Org.OpenAPITools.Client;
using Org.OpenAPITools.Model;

public class FindPetsByTagsDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            var response = new PetApi(config).FindPetsByTags(
                tags: null
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Pet#FindPetsByTags: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: examples/petstore/oseg_generated/php/FindPetsByTags_default_example.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPI\Client\Configuration::getDefaultConfiguration();

try {
    $response = (new OpenAPI\Client\Api\PetApi($config))->findPetsByTags(
        tags: null,
    );

    print_r($response);
} catch (OpenAPI\Client\ApiException $e) {
    echo 'Exception when calling Pet#findPetsByTags: '
        . print_r($e->getResponseObject());
}
